src/main/webapp/js/countryCodes.es6

```diff
@@ -1,269 +1,267 @@
-'use strict';
-
 // Country code data from:
 // https://gist.github.com/maephisto/9228207
 // https://github.com/michaelwittig/node-i18n-iso-countries/blob/master/codes.js
 
-var countryToCode = {
-    'Afghanistan': 'AFG',
+const countryToCode = {
```

Comment:
Could you disable the rule here and continue using quotes? This looks harder to maintain (can't use select all occurences, etc on IDEs).